source/extensions/common/wasm/stats_handler.cc

```diff
@@ -82,6 +82,14 @@ void LifecycleStatsHandler::onEvent(WasmEvent event) {
 
 int64_t LifecycleStatsHandler::getActiveVmCount() { return active_wasms; };
 
+// The custom namespace prefix which prepends the user-defined metrics.
+// Note that the prefix is removed from the final output of /stats endpoints.
+constexpr absl::string_view CustomStatNamespace = "wasmcustom";
+
+void ensureCustomStatNamespaceRegistered(Stats::CustomStatNamespaces& custom_namespaces) {
```

Comment:
nit: does this function layer make call-sites more readable compared to just inlining the contents? I think inlining would be more readable as it might fit on one line  :)

Reply:
inlined!